authentik/stages/authenticator_validate/models.py

```diff
@@ -30,6 +31,7 @@ def default_device_classes() -> list:
         DeviceClasses.WEBAUTHN,
         DeviceClasses.DUO,
         DeviceClasses.SMS,
+        DeviceClasses.EMAIL,
```

Comment:
Dunno if this should be selected by default

Reply:
in general yes